### DISCUSSION TITLE
Support for Json_schema_extra example data?

### DISCUSSION THREAD

**Discussion by robswc** [vitalik/django-ninja#883] [Q&A]
Hey all!

Might have missed it in the docs, but is there anyway to achieve this with django-ninja?

https://fastapi.tiangolo.com/tutorial/schema-extra-example/

Thanks!

**Comment by vitalik** [comment]
@robswc this should work with latest django ninja (`pip install django-ninja==1.0b2`)

**Reply by robswc** [reply]
Thank you for getting back to me!

I am on the latest version and have copied the example from the fastapi link.

I cannot seem to get it to load into the docs though.  Any ideas on how to troubleshoot?

Thanks!
